### PR TITLE
DEVCVL-226 - [FE] <Paginator /> Component

### PR DESCRIPTION
**Pull request by pracheebapate**
pagination-2022-04-13_19-10

### CODE REVIEW

**Review by pracheebapate — COMMENTED** [review]
Could not name the component as _Pagination_ because ant design is already using it.

File: src/components/pagination/CVLPagination.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { Pagination } from 'antd';
```
Comment (shakil-adv) [inline src/components/pagination/CVLPagination.js:2]: import { Pagination as AntPagination } from 'antd'
you can use alias import to solve name conflicts

Comment (pracheebapate) [reply src/components/pagination/CVLPagination.js]: Done. Thanks that was helpful.

**Review by kyawthetkt — APPROVED** [review]

File: src/components/pagination/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            total={total}
+            showSizeChanger={false}
+            showQuickJumper={false}
+            onChange={onChange}
+        />
+    )
+}
+
+Pagination.propTypes = {
+    defaultPageSize: PropTypes.number,
```
Comment (shakil-adv) [inline src/components/pagination/Pagination.js:30]: currently the proptypes says that the props are optional. If any prop is required, please add `.required` at the end. For example, defaultPageSize: PropTypes.number.required,

Comment (pracheebapate) [reply src/components/pagination/Pagination.js:30]: Thanks. Added 

> isRequired

**Review by shakil-adv — APPROVED** [review]

**Review by jibrail-idris — APPROVED** [review]

**Pull request merged**